content/posts/2022-01-11-recap-2021.md

```diff
@@ -0,0 +1,31 @@
+---
+title: CSSS 2021 Recap + 2022 Outlook
+date: 2022-01-11T18:48:36-08:00
+author: Eric Yan
+tags: [club-activities]
+categories: [services]
+images:
+    - /files/2022-01-11-Recap-2021.png
+---
+\
```

Comment:
For some reason, this post is not using our custom [layout](https://github.com/ubccsss/ubccsss.org/blob/master/layouts/events/content.html) but is instead using the bootstrap layout.